poetry/console/commands/new.py

```diff
@@ -46,13 +52,21 @@ def handle(self):
 
         readme_format = "rst"
 
-        config = GitConfig()
         author = None
-        if config.get("user.name"):
-            author = config["user.name"]
-            author_email = config.get("user.email")
+        author_name = self.option("author-name")
+        author_email = self.option("author-email")
+
+        if author_email and not author_name:
+            raise ValueError(
+                "`--author-name` with no default value not used when option `--author-email` "
+                "is defined or has a default value. "
+                "A value for `--author-name` is required when `--author-email` is defined."
+            )
+        if author_name:
             if author_email:
-                author += " <{}>".format(author_email)
+                author = "{name} <{email}>".format(name=author_name, email=author_email)
+            else:
+                author = author_name
```

Comment:
This will fail validation I think. We need both to construct a valid author right?

Reply:
I just saw your comment https://github.com/python-poetry/poetry/pull/2745#discussion_r485630817. Should be fine. I am wondering if PyPI likes it. Should probably check there too.

Reply:
I've looked at some documentation at various sites, but based on [packaging.python.org](https://packaging.python.org/specifications/core-metadata/#author), the author is
>A string containing the author’s name at a minimum; additional contact information may be provided.

Which makes me *think* PyPI would be OK with it.

But [Poetry's *own* documentation](https://python-poetry.org/docs/pyproject/#authors) states

>Authors must be in the form `name <email>`.

So I believe that I should make the author *always* be either `None` or `"Author Name <author@email.com>"` after all. Will push up a new commit.